pkg/taskrunmetrics/metrics.go

```diff
@@ -219,6 +224,11 @@ func viewRegister(cfg *config.Metrics) error {
 		Measure:     runningTRsThrottledByNodeCount,
 		Aggregation: view.LastValue(),
 	}
+	runningTRsWaitingOnBundleResolutionCountView = &view.View{
```

Comment:
Where is this being used?

Reply:
see below at https://github.com/tektoncd/pipeline/pull/7094/files#diff-7bb37a1fbfb2780494fcbb3f55f273ac112b0503172188595cf81c4fa8854848R245 and https://github.com/tektoncd/pipeline/pull/7094/files#diff-7bb37a1fbfb2780494fcbb3f55f273ac112b0503172188595cf81c4fa8854848R245

the associated metric is ultimately incremented at https://github.com/tektoncd/pipeline/pull/7094/files#diff-7bb37a1fbfb2780494fcbb3f55f273ac112b0503172188595cf81c4fa8854848R399

Reply:
Got it. I got confused due to the naming. 
Can you please rename it to `runningTRsWaitingOnTaskResolutionCountView` instead of `runningTRsWaitingOnBundleResolutionCountView`?

Reply:
update pushed @khrm